protoc-gen-debug/main.go

```diff
@@ -41,8 +42,10 @@ func main() {
 		log.Fatal("unable to write request to disk: ", err)
 	}
 
-	data, err = proto.Marshal(&plugin_go.CodeGeneratorResponse{})
-	if err != nil {
+	var supportedFeatures = uint64(pluginpb.CodeGeneratorResponse_FEATURE_PROTO3_OPTIONAL)
+	if data, err = proto.Marshal(&plugin_go.CodeGeneratorResponse{
```

Comment:
IIUC this again makes PGS by default claim to support `CodeGeneratorResponse_FEATURE_PROTO3_OPTIONAL`. My suggestion was to create a new `InitOption` type (in `init_option.go`), which will set the generator's persister's supportedFeatures variable. This gives full control to the user.

Reply:
see https://github.com/lyft/protoc-gen-star/blob/master/init_option.go#L43

Reply:
I've changed the behaviour in `init_option.go`, though i've left it as-is in `protoc-gen-debug` as the latter is a dev tool.

Reply:
Makes sense to me - but perhaps leave a comment to that effect.